snooty/n.py

```diff
@@ -512,3 +512,17 @@ class Transition(Node):
 class Table(Parent[Node]):
     __slots__ = ()
     type = "table"
+
+
+@dataclass
+class ComposableDirective(Directive):
+    __slots__ = "composable_options"
+    composable_options: List[
+        Dict[str, Union[str, List[Dict[str, str]], Dict[str, str]]]
+    ]
```

Comment:
Referring to this Union: I see examples in the tech spec of the `str`, I see examples of the `List[Dict[str, str]]`, but I don't see examples of the `Dict[str, str]`... Can you point that out? ...also I will probably see this as I go further into the review lol

Reply:
I saw it later in the review!